src/components/VotingResultTag/VotingResultTag.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+	import { DefaultVotingResult } from '$models/voting';
+	import { Tag } from 'carbon-components-svelte';
+
+	export let result: DefaultVotingResult | string;
+	export let isLarge = false;
+
+	let tagColor = 'bg-purple-70 text-text-04';
+
+	$: label = result;
```

Comment:
I am afraid that this line might make a race condition: both this and the switch case are reactive,  if this one is run after the switch (which we can never be sure of), then we will get an unexpected result for the `DefaultVotingResult` case.

Since we have `label = result;` on line 23 under the reactive switch, it will update the label when the `result` changes. So I suggest removing the reactivity on this line and just assigning it to an empty string like:

```ts
let label = '';
```

Reply:
Thank you. I will commit another change.